docs-2.0/14.client/6.nebula-go-client.md

```diff
@@ -0,0 +1,308 @@
+# Nebula Go
+
+Nebula Go是一款Go语言的客户端，可以连接、管理Nebula Graph图数据库。
+
+## 前提条件
+
+已安装Go，版本为1.13及以上。
+
+## 版本对照表
+
+|Nebula Graph版本|Nebula Go版本|
+|:---|:---|
+|{{ nebula.release }}|2.1.0|
+|2.0.1|2.0.0-GA|
+|2.0.0|2.0.0-GA|
+
+## 下载Nebula Go
+
+### 下载压缩包
+
+在[Nebula Go](https://github.com/vesoft-inc/nebula-Go/releases)下载页面，确认需要的版本，下载到机器上并解压缩。
+
+### 克隆源码
+
+克隆Nebula Go源码到机器。
+
+- （推荐）如果需要下载指定版本的Nebula Go，请使用选项`--branch`指定分支。例如安装v{{ nebula.release }}发布版本，请执行如下命令：
+
+  ```bash
+  $ git clone --branch v{{ nebula.release }} https://github.com/vesoft-inc/nebula-go.git
+  ```
+
+- 如果需要安装日常开发版本，请执行如下命令下载`master`分支的源码：
+
+  ```bash
+  $ git clone https://github.com/vesoft-inc/nebula-go.git
+  ```
```

Comment:
We don't really need this part. You only clone the source code when you want to modify it, and for most users, this is a rare scenario.

Reply:
It's optional, and ok